app/lib/bulk_download_service.rb

```diff
@@ -48,6 +48,12 @@ def self.generate_curl_configuration(study_files:,
       manifest_config += "output=\"#{study.accession}/file_supplemental_info.tsv\""
       curl_configs << manifest_config
     end
+    byebug
+    MetricsService.log('file-download:curl-config', {
+      num_files: study_files.count,
+      num_studies: studies.count,
+      study_accesions: studies.map(&:accession)
```

Comment:
Please update to use camelCase, which Bard uses for property names like these:

```suggestion
      numFiles: study_files.count,
      numStudies: studies.count,
      studyAccesions: studies.map(&:accession)
```

(The [lexicon](https://mixpanel.com/project/2085496/view/19055/app/lexicon#transformations/event-properties) shows some exceptions and variations, but camel case is the most common for app-defined properties.)